.github/workflows/docker.yml

```diff
@@ -3,13 +3,15 @@ name: Build and publish Forest Docker image
 on:
   push:
     branches: [main]
+    tags:
+    - '*'
   pull_request:
 
 jobs:
   build-and-push-docker-image:
     name: Build Docker image and push to repositories
     runs-on: buildjet-8vcpu-ubuntu-2204
```

Comment:
Do we need buildjet?

Reply:
A cold build takes upwards of 45 minutes on GitHub which is significantly slower than a cold build outside of docker. Not sure why. Maybe I'm doing something wrong and not allowing docker to use all the available cores.

Reply:
It's weird, not a blocker, though. We can try to get rid of this in another task.